modules/mssql/variables.tf

```diff
@@ -118,6 +118,14 @@ variable "database_flags" {
   default = []
 }
 
+variable "active_directory_config" {
+  description = "Active domain that the SQL instance will join."
+  type = list(object({
+    domain = string
+  }))
+  default = []
+}
```

Comment:
Can we just make this a string `active_directory_config_domain` since that is the only supported value?

Reply:
Gotcha, in that case lets use `map(string)` and use lookup for relevant values so its extensible? My reasoning for the ask is TF does not support defaults on objects and I want to minimize breaking changes to the module in the future.

Reply:
Sounds good, submitted the proposed change.